lib/src/firestore.dart

```diff
@@ -607,6 +640,13 @@ class GeoPoint {
   int get hashCode => hash2(latitude, longitude);
 }
 
+/// An immutable object representing an array of bytes.
+class Blob {
+  Blob(this.data);
+
+  Uint8List data;
+}
+
```

Comment:
Wondering if we can just use `Uint8List` directly without this wrapper? Or did you have more plans for this class?

Reply:
That's the main question. Currently I'm working on an abstraction of the firestore API. Most implementation have a Blob class (flutter: https://github.com/flutter/plugins/blob/master/packages/cloud_firestore/lib/src/blob.dart, web: https://github.com/firebase/firebase-dart/blob/master/lib/src/interop/firestore_interop.dart).

It is only the node API that does not have this concept since it directly handles ArrayBuffer (i think). Having a class allow helpers like you suggest and make it less confusing between a List<int> which is accepted as a firestore value (not as a blob) and Uint8List which can be seen as a blob in the console (cannot be edited though)